validator/project_validator.go

```diff
@@ -116,6 +119,16 @@ func (v ValidationErrors) HasError() bool {
 	return false
 }
 
+// Has returns if any of the errors are at the given level.
+func (v ValidationErrors) Has(level ValidationErrorLevel) bool {
+	for _, err := range v {
```

Comment:
nit: to reduce copy-pasting, might be simpler to just return `len(v.AtLevel(level)) > 0`.

Reply:
I redid it to the one you suggested! I don't think it'll be a bottleneck either but I have heard of the horror stories that some projects have hundreds of warnings 😨

Reply:
It's true that some projects have a lot of warnings and this is technically less efficient, but in practice, iterating in-memory through hundreds of items is so fast that it really won't make a difference. It's really only doing something expensive in a loop (e.g. DB calls in a loop) that needs optimization.